popup.html

```diff
@@ -5,7 +5,9 @@
       </style>
     </head>
     <body>
-      <input type="checkbox" id="active"/> Activate timer
+      <label for="reload-int-input">Refresh interval (seconds)</label>
+      <input type="number" min='1' id="reload-int-input" />
+      <input type="checkbox" id="active"/> Activate Auto-reload
```

Comment:
Along with the label that you created, can you please put  the "Activate Auto-reload" inside a label tag?

Reply:
Done